.github/workflows/main.yml

```diff
@@ -21,5 +21,8 @@ jobs:
       - name: Tslint check
         run: npm run tslint
 
+      - name: Typescript check
```

Comment:
In the upstream I can see `eslint` check. But it's missing here. https://github.com/WideChat/Apps.Dialogflow/compare/master...RocketChat:Apps.Dialogflow:master

Reply:
Should I remove tslint or keep both?

Reply:
just use eslint please and remove tslint